x-pack/plugins/reporting/server/browsers/chromium/driver/screenshot_stitcher/get_clips.test.ts

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { toArray } from 'rxjs/operators';
+import { $getClips } from './get_clips';
+import { Rectangle } from './types';
+
+function getClipsTest(
+  description: string,
+  input: { rectangle: Rectangle; max: number },
+  expectedClips: { clips: Rectangle[] }
+) {
+  test(description, async () => {
+    const clips = await $getClips(input.rectangle, input.max)
+      .pipe(toArray())
+      .toPromise();
+    expect(clips.length).toBe(expectedClips.clips.length);
```

Comment:
__question:__ can't we just replace this:
```js
expect(clips.length).toBe(expectedClips.clips.length);
for (let i = 0; i < clips.length; ++i) {
  expect(clips[i]).toEqual(expectedClips.clips[i]);
}
```
with that?
```js
expect(clips).toEqual(expectedClips.clips);
```

Reply:
Those two should be identical.